src/Post_Command.php

```diff
@@ -173,6 +180,21 @@ public function create( $args, $assoc_args ) {
 		$array_arguments = array( 'meta_input' );
 		$assoc_args      = \WP_CLI\Utils\parse_shell_arrays( $assoc_args, $array_arguments );
 
+            if( isset( $assoc_args['from-post'] ) ) {
+                  $post     = $this->fetcher->get_check( $assoc_args['from-post'] );
+                  $post_arr = get_object_vars( $post );
+                  $post_id  = $post_arr['ID'];
```

Comment:
`$post_arr['ID']` should be unset, correct?

Reply:
yes, you are right. Actually, if we don't unset id it will still generate new id. But, for a safer side, I will correct.